.github/workflows/ci-pipeline.yaml

```diff
@@ -48,6 +48,6 @@ jobs:
         with:
           go-version: ${{ env.GO_VERSION }}
       - name: Go tests
-        run: go test -v -race -coverprofile=coverage.out -covermode=atomic ./...
+        run: sudo go test -v -race -coverprofile=coverage.out -covermode=atomic ./...
```

Comment:
💡Thank you for this. Archiving only: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#administrative-privileges - if anyone ponders `sudo` use.

minor small question is: Why is `sudo` needed?